apps/server/src/shared/testing/factory/external-class-dto.factory.ts

```diff
@@ -0,0 +1,11 @@
+import { ExternalClassDto } from '@modules/provisioning';

```

Comment:
There is a linter warning here. If the factory is moved to the module where the corresponding class is located, this should not occur. As an example, you can take the school module, in which there is a 'testing' folder with the corresponding factories.